pkgs/toshimaru/nyan/registry.yaml

```diff
@@ -0,0 +1,16 @@
+packages:
+  - type: github_release
+    repo_owner: toshimaru
+    repo_name: nyan
+    description: Colored cat command which supports syntax highlighting
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: semver("<= 0.10.16")
+        no_asset: true
+      - version_constraint: "true"
+        asset: nyan_{{.OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        supported_envs:
+          - linux
+          - darwin
+        rosetta2: true
```

Comment:
I think `rosetta2` isn't necessary at the latest version.

https://github.com/toshimaru/nyan/releases/tag/v1.0.10

[nyan_Darwin_arm64.tar.gz](https://github.com/toshimaru/nyan/releases/download/v1.0.10/nyan_Darwin_arm64.tar.gz)

Reply:
cmdx s `toshimaru/nyan` generates the different code.
Please follow the contribution guide as you checked the check list.

Reply:
I apologize for skipping over the important chapter about contributing. Thank you for pointing it out and for handling the corrections until the merge. As a user, I'm delighted that such a fantastic tool(toshimaru/nyan) has been added to the aqua-registry.